go/consensus/tendermint/abci/mux.go

```diff
@@ -221,11 +232,23 @@ func (a *ApplicationServer) Pruner() StatePruner {
 }
 
 // SetEpochtime sets the mux epochtime.
-// XXX: epochtime needs to be set before mux can be used.
+//
+// Epochtime must be set before the multiplexer can be used.
 func (a *ApplicationServer) SetEpochtime(epochTime epochtime.Backend) {
 	a.mux.state.timeSource = epochTime
 }
 
+// SetFeeHandler configures the transaction fee handler for the
+// ABCI multiplexer.
+func (a *ApplicationServer) SetTransactionFeeHandler(handler TransactionFeeHandler) error {
+	if a.mux.state.txFeeHandler != nil {
+		return fmt.Errorf("mux: transaction fee handler already configured")
```

Comment:
similar check might make sense above for `SetEpochtime` as well :)

Reply:
Done.